### DISCUSSION TITLE
Instructions on using Arduino_GFX with the new Lilygo AMOLED device

### DISCUSSION THREAD

**Discussion by 3Dsf-info** [moononournation/Arduino_GFX#313] [Show and tell]
Hi, and thanks for all your help in getting the code to work!

I've written up step by step instructions on how to use Arduino_GFX to display simple animations on the new high-resolution Lilygo board. Hopefully it'll be useful for other people.

https://sites.google.com/site/3dsfinfo/mini-displays

**Comment by 3Dsf-info** [comment]
Also sent you a donation.

**Comment by moononournation** [comment]
Your link is broken, but I think it is RM67162 and it is already supported.

**Comment by 3Dsf-info** [comment]
Yes - the URL is updated and is now:

https://sites.google.com/site/3dsfinfo/lights-for-model-makers-technology-and-techniques/high-rez-microcontroller-displays
